packages/oui-button/button.less

```diff
@@ -63,6 +63,12 @@
     &.@{oui-button-selector}_icon-right {
       #oui > .button-full-width-icon-right(@oui-icon-selector);
     }
+
+    &.@{oui-button-selector} {
```

Comment:
Useless selector, you can put directly `.oui-icon-external_link { }` in the parent selector